src/app/content/studyGuides/components/Filters.tsx

```diff
@@ -90,7 +92,8 @@ const ConnectedPrintButton = connect(
 
 export default () => {
   const userLoggedOut = useSelector(loggedOut);
-  const [isUTGopen, setUTGopen] = React.useState(false);
+  const showUTGInitially = React.useMemo(() => !Cookies.get(cookieUTG), []);
+  const [isUTGopen, setUTGopen] = React.useState(showUTGInitially);
```

Comment:
Combined with the comment about refactoring this logic: 
- remove `showUTGInitially`
- do `React.useState(!Cookies.get(cookieUTG)) instead.